gnucash/import-export/ofx/gnc-ofx-import.c

```diff
@@ -1069,12 +1068,13 @@ void gnc_file_ofx_import (GtkWindow *parent)
     gtk_file_filter_add_pattern (filter, "*.[oqOQ][fF][xX]");
     filters = g_list_prepend( filters, filter );
 
-    selected_filename = gnc_file_dialog(parent,
-                                        _("Select an OFX/QFX file to process"),
-                                        filters,
-                                        default_dir,
-                                        GNC_FILE_DIALOG_IMPORT);
+    selected_filenames = gnc_file_dialog_multi (parent,
+                                               _("Select one or multiple OFX/QFX file(s) to process"),
+                                               filters,
+                                               default_dir,
+                                               GNC_FILE_DIALOG_IMPORT);
     g_free(default_dir);
+    selected_filename = selected_filenames->data;
 
     if (selected_filename != NULL)
```

Comment:
Change this to..
    if (g_slist_length (selected_filenames) > 0)

Reply:
An empty GSlist is a `nullptr`, so no point in counting the list. `if (selected_filenames)` does everything needed with the least ceremony.

Reply:
@jeanlaroche Did you see John's suggestion, it's simpler and better than my change.

Reply:
Yep, saw that. Will change it. :)